scvi/model/gimvi.py

```
@@ -73,16 +73,29 @@ def __init__(
         super(GIMVI, self).__init__(use_cuda=use_cuda)
         self.use_cuda = use_cuda and torch.cuda.is_available()
         self.adatas = [adata_seq, adata_spatial]
+
         seq_var_names = _get_var_names_from_setup_anndata(adata_seq)
         spatial_var_names = _get_var_names_from_setup_anndata(adata_spatial)
+
+        assert set(spatial_var_names) <= set(seq_var_names), ValueError(
```

let's raise an error here